examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -0,0 +1,319 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+enum _Location {
+  Barbados,
+  Bahamas,
+  Bermuda
+}
+
+const TextStyle text54 = const TextStyle(
+  color: Colors.black54,
+  fontSize: 15.0
+);
+const TextStyle text87 = const TextStyle(
+  color: Colors.black87,
+  fontSize: 15.0
+);
+
+class DualHeaderWithHint extends StatelessWidget {
+  DualHeaderWithHint({
+    this.name,
+    this.value,
+    this.hint,
+    this.showHint
+  });
+
+  final String name;
+  final String value;
+  final String hint;
+  final bool showHint;
+
+  Curve _flipCurves(bool isExpanded) {
+    return isExpanded ? new Interval(0.0, 0.6, curve: Curves.ease) :
+        new Interval(0.4, 1.0, curve: Curves.ease);
+  }
+
+  Widget _crossFade(Widget first, Widget second, bool isExpanded) {
```

Comment:
@Hixie I asked myself the same question.

The widget is private, so you don't have access to it.